src/main/java/appeng/services/version/ModVersionFetcher.java

```diff
@@ -30,11 +53,18 @@ public Version get()
 		{
 			return new DoNotCheckVersion();
 		}
-		else
+
+		try
 		{
 			final Version version = this.parser.parse( this.rawModVersion );
 
 			return version;
 		}
+		catch( final Exception e )
```

Comment:
Should be probably exported to a new class handling that. Thus can also make it testable.